themes/digital.gov/layouts/shortcodes/img.html

```diff
@@ -76,7 +76,7 @@
         {{- end -}}
         "
       {{- end -}}
-      sizes="(max-width: 800px) 100vw, 400px"
+      sizes="(max-width: 800px) 100vw, 1200px"
```

Comment:
When on mobile, we are loading `1200`. We need to add other breakpoint sizes to load the appropriate image size depending on the device/viewport.

See the documentation for the [`sizes` attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images).

We should at least account for mobile, tablet and desktop. See [USWDS sizes](https://designsystem.digital.gov/utilities/layout-grid/#utilities-settings) to update the responsive image component.